src/server.rs

```diff
@@ -52,7 +52,8 @@ use crate::{
 };
 
 const API_PREFIX: &str = r#"/api/v1/task"#;
-const TEXT_API_PREFIX: &str = r#"/api/v1/text/task"#;
+// New orchestrator API
+const TEXT_API_PREFIX: &str = r#"/api/v2/task"#;
```

Comment:
@gkumbhat Are we dropping modality from endpoints?

Reply:
woops. wrong thing got removed here. We are dropping `task` not `text`.. I'll fix it. Thanks for catching!

Reply:
fixed!